internal/pkg/argocdapp/argocdapp.go

```diff
@@ -42,3 +42,13 @@ func kubectlApply(file string) {
 	}
 	log.Println(string(strings.TrimSuffix(string(stdout), "\n")))
 }
+
+func kubectlDelete(file string) {
```

Comment:
kubectlDelete() may be failed. And it needed return the error massage. But I think it’s fine if you don’t consider this problem now, because the previous kubectlApply() function is the same.

Reply:
return info added